<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover{color: blue; font: bold 10pt;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_load">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Загрузить картинку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить картинку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_encode">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Закодировать текст</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_decode">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Декодировать текст</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>381</width>
      <height>211</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите текст</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;span style=&quot;font-size:22pt; color:#0000ff;&quot;&gt;&lt;b&gt; Стеганография &lt;/b&gt;&lt;/span&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt; Загрузите файл.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Доступно:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Всего:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ftotal">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="ffree">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="fhave">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Содержит:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Загруженный файл:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущее сообщение:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Содержит:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Доступно:</string>
    </property>
   </widget>
   <widget class="QLabel" name="inp_size">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="inp_free">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
